TF_Service_dotNet/TouchFree Service/Program.cs

```diff
@@ -11,6 +12,8 @@ static void Main(string[] args)
             UpdateBehaviour updateLoop = new();
             updateLoop.OnUpdate += TickTock;
 
+            TrackingConnectionManager trackingConnectionManager = new TrackingConnectionManager();
```

Comment:
I think this will collide with my other PR as we'll need to instance this as an ASP Service. Not an issue but we should note that whichever gets merged second will need to handle the changes from the first.